### DISCUSSION TITLE
Async version of vcf::reader::Builder?

### DISCUSSION THREAD

**Discussion by holtgrewe** [zaeleus/noodles#209] [Q&A] [answered]
Is there a version of `vcf::reader::Builder` that can be used to create an `vcf::AsyncReader`?

I need to perform I/O to S3 and all the interfaces are async so support for this would be greatly appreciated.

**Comment by zaeleus** [comment, marked as answer]
Reader builders in noodles are mainly a convenience around opening local files. In the case of using an async stream, I'm not sure a builder would be useful here, as building from a reader simply calls the reader constructor (`AsyncReader::new`).
